rueidisotel/trace.go

```diff
@@ -61,6 +61,30 @@ func WithDBStatement(f StatementFunc) Option {
 // StatementFunc is a the function that maps a command's tokens to a string to put in the db.statement attribute
 type StatementFunc func(cmdTokens []string) string
 
+type commandMetrics struct {
+	duration   metric.Float64Histogram
+	errors     metric.Int64Counter
+	addOpts    []metric.AddOption
+	recordOpts []metric.RecordOption
+}
+
+func (c commandMetrics) recordDuration(ctx context.Context, op string) func() {
+	now := time.Now()
+	return func() {
+		opts := append(c.recordOpts, metric.WithAttributes(attribute.String("operation", op)))
```

Comment:
The `operation` attribute can create an unlimited number of dimensions on the metric, which could potentially overwhelm client memory. I think we can't include the `operation` attribute by default.

Reply:
Would coalescing the multi commands into a single `"multi"` help? I was only aware of a limited (though high) number of supported commands, do any take user input?

Reply:
I noticed you added a toggle, so there’s no need to combine multiple commands into a single “multi”.

Reply:
Got it, updated!